CMakeLists.txt

```diff
@@ -184,6 +184,7 @@ option(THEROCK_ENABLE_COMM_LIBS "Enable building of comm libraries" "${THEROCK_E
 option(THEROCK_ENABLE_MATH_LIBS "Enable building of math libraries" "${THEROCK_ENABLE_ALL}")
 option(THEROCK_ENABLE_ML_LIBS "Enable building of ML libraries" "${THEROCK_ENABLE_ALL}")
 option(THEROCK_ENABLE_PROFILER "Enable building the profiler libraries" "${THEROCK_ENABLE_ALL}")
+option(THEROCK_ENABLE_DC_TOOLS "Enable building of data center tools" "${THEROCK_ENABLE_ALL}")
```

Comment:
Let's land this with `OFF` instead of `${THEROCK_ENABLE_ALL}`. Then you can follow-up by moving the grpc archive from a git recursive fetch to a tarball on S3. Then enable the flag here to fully launch it.

I would just like you to be able to lock in your change in a way that is non impacting until ready to be enabled.